### ISSUE TITLE
Admin Console Feedback - clients/list

### ISSUE THREAD

**Issue by stianst** [keycloak/keycloak-community#92]
https://www.keycloak.org/keycloak-community/design/admin-console/#/clients/list

**Comment by pcfleischer** [comment]
I like the new columns but I noticed the enabled column is gone (may have been gone in previous versions), would it be possible to use an overlay to indicate enabled/disabled instead?

**Comment by dasunsrule32** [comment]
> @pcfleischer
> I like the new columns but I noticed the enabled column is gone (may have been gone in previous versions), would it be possible to use an overlay to indicate enabled/disabled instead?

I agree, it's nice to be able to do a really quick scan and see which clients are enabled.

**Comment by yih-wang** [comment]
> @dasunsrule32
> > @pcfleischer
> > I like the new columns but I noticed the enabled column is gone (may have been gone in previous versions), would it be possible to use an overlay to indicate enabled/disabled instead?
> 
> I agree, it's nice to be able to do a really quick scan and see which clients are enabled.

Right, we are trying to remove the enable column in this new version. It's good to know that it is a widely used feature. We will figure out how to show it in this new layout and add it in our next updates.

**Comment by velias** [comment]
> @pcfleischer
> I like the new columns but I noticed the enabled column is gone (may have been gone in previous versions), would it be possible to use an overlay to indicate enabled/disabled instead?

Agree, from my admin experience "enabled" status is rather important info which should be indicated somehow. Or/and you should allow filtering over this state also.

**Comment by stefanwendelmann** [comment]
A Checkbox on the left side for bulk actions?

**Comment by yih-wang** [comment]
> A Checkbox on the left side for bulk actions? And bulk actions for the User Page would be nice too.

For now, we don't support any bulk actions in Clients. But we are likely going to support bulk deleting in the Users list. Can I ask what bulk actions you are expecting from the Clients list and Users list?

**Comment by yih-wang** [comment]
> @stefanwendelmann
> A Checkbox on the left side for bulk actions? And bulk actions for the User Page would be nice too. 

For now, we don't support any bulk actions in Clients. But we are likely going to support bulk deleting in the Users list. Can I ask what bulk actions you are expecting from the Clients list and Users list?

**Comment by akashsethiya** [comment]
> @yih-wang
> > @stefanwendelmann
> > A Checkbox on the left side for bulk actions? And bulk actions for the User Page would be nice too. 
> 
> For now, we don't support any bulk actions in Clients. But we are likely going to support bulk deleting in the Users list. Can I ask what bulk actions you are expecting from the Clients list and Users list?

Adding or removing users to a group in bulk, granting or revoking roles

**Comment by yih-wang** [comment]
@akashsethiya The functions you mentioned have already been added to the new design. You can check it out after we update the Users and Groups new design to the website.

**Comment by Rjvs** [comment]
> Right, we are trying to remove the enable column in this new version.

Why would you want to do that?  It's one of the most important pieces of information about a client.

**Comment by stianst** [comment]
> > Right, we are trying to remove the enable column in this new version.
> 
> Why would you want to do that? It's one of the most important pieces of information about a client.

I think it's important to display somehow, but it does not warrant using a whole column. I would imagine in most cases there's very few if any disabled clients. Not sure why you would really have a disabled client.

**Comment by Rjvs** [comment]
> > we are trying to remove the enable column in this new version.
>  it's important to display somehow, but it does not warrant using a whole column. I would imagine in most cases there's very few if any disabled clients. Not sure why you would really have a disabled client.

I guess it could be greyed out or something.  We aren't in production yet but our use-case is disabling clients that the customer isn't using; we provide a single distribution and just disable the clients that they haven't licensed.

**Comment by akashsethiya** [comment]
a search option in the realms list is much needed. any plans for this?

**Comment by yih-wang** [comment]
@akashsethiya Yes. We have added the search function in the new realm selector. Will upload the design when everything is done.
